app/src/main/java/fr/neamar/kiss/broadcast/NewAppInstalledHandler.java

```diff
@@ -18,6 +21,14 @@
 
     @Override
     public void onReceive(Context ctx, Intent intent) {
+
+        if ("android.intent.action.PACKAGE_ADDED".equals(intent.getAction())) {
+            //add new package to history
+            String packageName = intent.getData().getSchemeSpecificPart();
+            String className = ctx.getPackageManager().getLaunchIntentForPackage(packageName).getComponent().getClassName();
```

Comment:
this can be null (for instance when installing a theme), maybe check for nullity before inserting into history?